ctre/basemotorcontroller.py

```diff
@@ -93,18 +96,40 @@ def set(self, mode: ControlMode, demand0: float, demand1: float = 0.0):
             In :attr:`.ControlMode.Position` mode, output value is in encoder ticks or an analog value, depending on the sensor.
             
             In :attr:`.ControlMode.Follower` mode, the output value is the integer device ID of the talon to duplicate.
-        :type demand0: float
         :param demand1:
             Supplemental value.  This will also be control mode specific for future features.
-        :type demand1: float
-
-        see :meth:`.selectProfileSlot` to choose between the two sets of gains.
+        :param demand1Type:
+            The demand type for demand1.
+            
+            * Neutral: Ignore demand1 and apply no change to the demand0 output.
+            * AuxPID: Use demand1 to set the target for the auxiliary PID 1.
+            * ArbitraryFeedForward: Use demand1 as an arbitrary additive value to the
+              demand0 output.  In PercentOutput the demand0 output is the motor output,
+              and in closed-loop modes the demand0 output is the output of PID0.
+    
+        Arcade Drive Example::
+        
+            _talonLeft.set(ControlMode.PercentOutput, joyForward, DemandType.ArbitraryFeedForward, +joyTurn)
+            _talonRght.set(ControlMode.PercentOutput, joyForward, DemandType.ArbitraryFeedForward, -joyTurn)
```

Comment:
I think the arguments here are the wrong way around. Or maybe we should have it this way around in the signature — I don't think it'd make sense to not specify the type of demand1 if you're passing it in anyway.